openapi/components/schemas/StorefrontQuoteChangeOrder.yaml

```
@@ -0,0 +1,474 @@
+            description: Amount of the invoice.
+            type: number
+            x-type: Money
+            format: double
+          subtotalAmount:
+            description: Subtotal amount of the invoice.
+            type: number
+            x-type: Money
+            format: double
+          discountAmount:
+            description: Discount amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+          shippingAmount:
+            description: Shipping amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+          taxAmount:
+            description: Tax amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+      items:
+        type: array
+        description: Invoice items.
+        items:
+          type: object
+          properties:
+            quoteItemId:
+              description: ID of the related quote item.
+              type: string
+              example: qti_0YV7DES3WPC5J8JD8QTVNZBZNZ
+            type:
+              type: string
+              description: Type of the invoice item.
+              x-basic: true
+              enum:
+                - debit
+                - credit
+            name:
+              description: Name of the invoice item.
+              type: string
+              maxLength: 1000
+            description:
+              description: Description of the invoice item.
+              type: string
+              example: Charge per approved transaction with DCC
+              maxLength: 255
+            priceDescription:
+              type: string
+              description: Price description of the invoice item.
+              example: 50% of the markup for approved transactions
+              maxLength: 255
+            unitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            quantity:
+              description: Quantity of the invoice item.
+              type: integer
+            period:
+              description: Date interval of the invoice item.
+              type:
+                - 'string'
+                - 'null'
+            setupUnitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialUnitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialPeriod:
+              description: Date interval of the invoice item trial.
+              type:
+                - 'string'
+                - 'null'
+            taxAmount:
+              description: Invoice item tax.
```

suggestion: for readability
```suggestion
              description: Tax amount of the invoice item.
```